app.js

```diff
@@ -1,88 +1,92 @@
 'use strict';
 const app = require('express')()
-const config = require('./lib/config')
+const find = require('lodash/find')
+const config = require('./lib/ConfigProvider')
 const HistoricDataProvider = require('./lib/HistoricDataProvider')
+const { gdax } = require('./lib/gdax')
+const strategy = require('./lib/Strategy')()
+const manager = require('./lib/PortfolioManager')
+const { getTrader, TraderBot } = require('./lib/TraderBot')
+const FeedService = require('./lib/FeedService')
+const configRoute = require('./api/routes/config');
 
-const { gdax, DANGER_LIVE_GDAX_DANGER } = require('./lib/gdax')
+// connect routes
+app.use('/config', configRoute);
 
-const Strategy = require('./lib/Strategy')
-const PortfolioManager = require('./lib/PortfolioManager')
-const TraderBot = require('./lib/TraderBot')
-const FeedService = require('./lib/FeedService')
+// catch 404 and forward to error handler
+app.use(function (req, res, next) {
+	var err = new Error('Not Found');
+	err.status = 404;
+	next(err);
+});
 
-let Spinner = require('cli-spinner').Spinner
-let spinner = new Spinner('working... %s')
```

Comment:
@kylerberry Getting rid of the spinner?

Reply:
yea, it never really worked reliably and now that we're moving toward a Web UI view I don't think it's necessary